ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -189,7 +189,7 @@ const BaseLocusListLabels = React.memo(({
       paLocusList,
       geneSymbol,
     }) : {
-      description: label,
+      description: locusListDescription,
```

Comment:
this was actually an existing bug where we were not showing the correct description for gene lists in either the geen or variant labels